services/spiget/spiget-download-size.service.js

```diff
@@ -0,0 +1,46 @@
+'use strict'
+
+const { BaseSpigetService, documentation } = require('./spiget-base')
+
+module.exports = class SpigetDownloadSize extends BaseSpigetService {
+  static get route() {
+    return {
+      base: 'spiget/download-size',
+      pattern: ':resourceid',
+    }
+  }
+
+  static get defaultBadgeData() {
+    return {
+      label: 'size',
+      color: 'brightgreen',
```

Comment:
the `brightgreen` color is typically used for badges to indicate some kind of success (and red for failures, etc.) for informational badges like this I think we should probably use something like `blue`